web-client/src/presenter/state.ts

```diff
@@ -640,6 +641,9 @@ export const baseState = {
   createOrderSelectedCases: [] as any[],
   currentJudges: [],
   currentPage: 'Loading',
+  currentTrialSessionLocation: undefined as
+    | TrialSessionLocationInfo
+    | undefined,
```

Comment:
Would you be open to wrapping this and `updatedTrialSessionLocation` in an object to avoid having two miscellaneous but closely coupled pieces of state in the root of the state tree? Also, do we ever unset this state, and is it worth doing? (Unlikely to be a big deal. Mostly just worried about continuing the pattern of not clearing state since that has caused problems in other cases.)